<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>499</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>352</width>
    <height>442</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PrimHesapla | CGB</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../prim.png</normaloff>../../../../prim.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="hesaplaButon">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hesapla!</string>
    </property>
   </widget>
   <widget class="QLabel" name="gelirLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Şube Gelirini Giriniz: (TL)</string>
    </property>
   </widget>
   <widget class="QLabel" name="giderLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Şube Giderini Giriniz (TL)</string>
    </property>
   </widget>
   <widget class="QLabel" name="aylikCironuz">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Aylık Cironuz: (TL) </string>
    </property>
   </widget>
   <widget class="QLabel" name="calisanlaraVerilecekPrimler">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Çalışanlara Verilecek Primler: (TL)</string>
    </property>
   </widget>
   <widget class="QLabel" name="MagazaMuduru">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mağaza Müdürü: (TL)</string>
    </property>
   </widget>
   <widget class="QLabel" name="MudurYardimcisi">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mağaza Müdür Yardımcısı: (TL)</string>
    </property>
   </widget>
   <widget class="QLabel" name="kasiyer">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kasiyer: (TL)</string>
    </property>
   </widget>
   <widget class="QLabel" name="satisDanismani">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Satış Danışmanı: (TL)</string>
    </property>
   </widget>
   <widget class="QLabel" name="ciroLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="mdrLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="mdyLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="ksyrLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="sdLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gelirText">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="giderText">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="satisDanismani_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Satış Danışmanı: (TL)</string>
    </property>
   </widget>
   <widget class="QLabel" name="satisDanismani_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Satış Danışmanı: (TL)</string>
    </property>
   </widget>
   <widget class="QLabel" name="sdLabel_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="sdLabel_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>477</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHakk_nda">
    <property name="title">
     <string>Hakkında</string>
    </property>
    <addaction name="actionProgram_bilgisi"/>
   </widget>
   <widget class="QMenu" name="menuYard_m">
    <property name="title">
     <string>Yardım</string>
    </property>
   </widget>
   <addaction name="menuHakk_nda"/>
   <addaction name="menuYard_m"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionProgram_bilgisi">
   <property name="text">
    <string>Program bilgisi</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
